src/ui/QuerySuggestPreview/QuerySuggestPreview.ts

```diff
@@ -32,23 +33,20 @@ export class QuerySuggestPreview extends Component implements IComponentBindings
   };
 
   static options: IQuerySuggestPreview = {
```

Comment:
Don't forget to add the following comment:

```
/**
 * @componentOptions
 */
```

to expose the options in the public documentation.

Reply:
Woops, I didn't know about this. I'll add that to both this PR and the documentation PR.